src/Ng/Jobs/Monitoring2MonitoringJob.cs

```diff
@@ -4,9 +4,11 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
```

Comment:
Are these necessary changes?

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/81d386ceb95e2f5839d6e63029ee23ca1b22326b